src/main/java/htsjdk/samtools/SamReaderFactory.java

```diff
@@ -57,6 +58,13 @@
     
     abstract public SamReader open(final File file);
 
+    public SamReader open(final Path path) {
```

Comment:
The reason I didn't add an abstract method here was to maintain backwards compatibility. Is there a better way of adding the overloaded method?